images/android-ndk-r22-jdk17-ubuntu24

```diff
@@ -0,0 +1,128 @@
+FROM amd64/ubuntu:noble-20240904.1
+
+# Avoid attempting to open dialog boxes
+ENV DEBIAN_FRONTEND="noninteractive"
+
+# Additional filesystems
+ADD filesystem/libsysconfcpus.tar.gz /
+
+# Install base packages
+RUN set -eu \
+ && apt-get update \
+ && apt-get -y install sudo git curl python3 python-is-python3 wget make python3-pip zip zlib1g \
+ && apt-get -y install software-properties-common \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN set -eu \
+ && apt-get update \
+ && apt-get -y install unzip ccache openjdk-11-jdk-headless apt-transport-https software-properties-common \
+ && rm -rf /var/lib/apt/lists/*
+
+# Install CLI tools for CI scripting
+RUN (type -p wget >/dev/null || (sudo apt update && sudo apt-get install wget -y)) \
+	&& sudo mkdir -p -m 755 /etc/apt/keyrings \
+	&& wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+	&& sudo chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+	&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+	&& sudo apt update \
+	&& sudo apt install jq gh -y
+
+# Install gcloud for Firebase testing
+RUN set -eu \
+  && echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
+  && wget -O - https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - \
+  && apt-get update -y \
+  && apt-get install google-cloud-sdk -y \
+  && apt-get autoremove --purge -y && apt-get clean
+
+# Install libffi for sdk-cicd
+RUN set -eu \
+  apt-get update -y && apt-get -y install libffi-dev
```

Comment:
`sdk-cicd` requires FFI library - on Ubuntu 24.04 that's https://packages.debian.org/buster/libffi-dev